docs/internet-identity-spec.adoc

```diff
@@ -197,6 +197,8 @@ type DeviceData = record {
   pubkey : DeviceKey;
   alias : text;
   credential_id : opt CredentialId;
+  purpose : { #recovery; #authentication };
+  key_type : { #unknown; #platform; #cross_platform; #seed_phrase };
```

Comment:
Is this actually valid Candid? If yes, what's the difference in using `variant` instead? Saving on representation bytes?

Reply:
It's not. Sorry I just used the Motoko syntax for these variants, but I see you fixed it in your backend PR so we can just merge it with that, right?